packages/rn-tester/RNTester/Info.plist

```diff
@@ -61,6 +61,8 @@
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 	</array>
+	<key>CADisableMinimumFrameDurationOnPhone</key>
+	<true/>
```

Comment:
is this something we should add to the template too?

Reply:
Yes, only with this we are able to run at 120fps. I'll create a followup once this PR gets merged.

Reply:
Would it be okay if we add `CADisableMinimumFrameDurationOnPhone` flag to the template without waiting for this PR? Without it, all Animated and Reanimated animations are capped at 60 fps, see https://github.com/software-mansion/react-native-reanimated/issues/7984#issuecomment-3148775591

Also, this flag is also enabled out-of-the-box in Expo app template.

Reply:
I think we can, if Expo is already enabling it by default, we can do it in the template.
It will be shipped in either 81.1 or 82, though.

Reply:
Cool! Do you want to submit the PR do you want me to do that?

Reply:
if you can do it, that would be great. Ideally, we should update the cocoapods infra to set that for all the projects, so users don't have to do anything on their side.

We would need cherry pick requests for all the supported versions @tomekzaw